interactive-shellbot.js

```diff
@@ -0,0 +1,29 @@
+/*
+# RUN THE BOT:
+  Get a Bot token from Slack:
+    -> http://my.slack.com/services/new/bot
+
+  Run your bot from the command line:
+    token=<MY TOKEN> API_AI_TOKEN=<MY AI TOKEN> node interactive-shellbot.js
+
+# EXTEND THE BOT:
+  Botkit has many features for building cool and useful bots!
+  Read all about it here:
+    -> http://howdy.ai/botkit
+*/
+'use strict';
+
+// Interactive shell for bot
```

Comment:
Yeah so the command to start it becomes the same for both production and development. Or I guess we could also just add another script to the `package.json` to enable running with `npm run shell`